httpie/core.py

```diff
@@ -108,19 +108,23 @@ def program(args, env, log_error):
             responses = [final_response]
 
         for response in responses:
-
             if args.check_status or downloader:
-                has_branched('program', 4)
+                if args.check_status:
+                    has_branched('program', 4)
+                else:
```

Comment:
Is this else-clause here because of the "add else-clauses to if-statements if they don't have an else-statement" criteria? Same goes for the stand alone else-clauses below.